co-authors-plus.php

```diff
@@ -380,14 +383,15 @@ public function coauthors_meta_box( $post ) {
 				<?php
 				foreach ( $coauthors as $coauthor ) :
 					$count++;
+					$avatar_url = get_avatar_url( $coauthor->ID, array( 'default' => 'gravatar_default' ) );
 					?>
 					<li>
-						<?php echo get_avatar( $coauthor->user_email, $this->gravatar_size ); ?>
```

Comment:
Should we keep the `get_avatar()` being `echo`'d because of `filter_get_avatar` already existing in CAP and incorporate the logic of the changes in that function? https://github.com/Automattic/Co-Authors-Plus/blob/f43f5034816ba59a60edb4abe85da0af13eb1912/php/class-coauthors-guest-authors.php#L1428